sdk/monitor/monitor-opentelemetry/src/shared/logging/logger.ts

```diff
@@ -60,26 +60,24 @@ export class Logger {
       suppressOverrideMessage: true,
     });
 
-    let azureLogLevelEnv =
-      process.env.APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL || process.env.AZURE_LOG_LEVEL;
-    let azureLogLevel: AzureLogLevel = "warning"; // default
+    const azureLogLevelEnv = process.env.APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL;
     switch (azureLogLevelEnv) {
       // Application Insights levels
       case "VERBOSE":
-        azureLogLevel = "verbose";
+        setLogLevel("verbose");
         break;
       case "INFO":
-        azureLogLevel = "info";
+        setLogLevel("info");
         break;
       case "WARN":
-        azureLogLevel = "warning";
+        setLogLevel("warning");
         break;
       case "ERROR":
-        azureLogLevel = "error";
+        setLogLevel("error");
+        break;
+      default:
+        setLogLevel(process.env.AZURE_LOG_LEVEL || "warning");
```

Comment:
seems that the compiler isn't happy about this line now, might have to do something like 

```suggestion
        setLogLevel(process.env.AZURE_LOG_LEVEL as AzureLogLevel || "warning");
```